### DISCUSSION TITLE
Miranda Detonate Arrows Crashes Devbuild

### DISCUSSION THREAD

**Discussion by ironorr84** [Legend-of-Dragoon-Modding/Severed-Chains#1744] [General]
Just like the title says. If I equip Miranda with Detonate Arrows, enter a battle, and try to attack, the game immediately crashes. If I switch it to Arrow of Force, it works just fine. When I switch back to Detonate Arrows, it crashes.

I have even played with the GameState Editor, I have sold the old Detonate Arrows, and then generated new ones from the Editor. Still the same crash.

**Comment by LordMonoxide** [comment]
What build are you on? I fixed this on sept 23
